### DISCUSSION TITLE
No results found on Kibana with Elastic Agent

### DISCUSSION THREAD

**Discussion by Snack-K** [pfelk/pfelk#451] [Q&A]
Hi everyone!

I have tried to setup the pfsense Elastic Agent with Fleet-server to receive OPNsense logs. However, I could not locate any logs from it. 

![pfsense_kibana](https://user-images.githubusercontent.com/105493284/203293624-e53dff4e-9fef-42f2-a46f-4bff30c081b0.png)

Here is what I set up:

![pfsense_agent_config](https://user-images.githubusercontent.com/105493284/203293845-46c02f54-cd59-4784-9522-0baf310f3295.png)

![OPNsense_config](https://user-images.githubusercontent.com/105493284/203293861-0b53a740-21cb-47b1-8d56-4df51c9742d3.png)

![netstat](https://user-images.githubusercontent.com/105493284/203293882-cdb05ed5-7fbd-4c68-a162-f68ca3980b74.png)

Here is the Agent log:

![Agent_log](https://user-images.githubusercontent.com/105493284/203294804-95eea892-b268-4857-baef-efeb30441498.png)

Can anyone help me with this?

**Comment by a3ilson** [comment]
@Snack-K - any luck?

The elastic agent is largely based on this GitHub repository. However, being able to amend and tweak files is not easily assessable...for that reason I have not utilized it.
